modules/appeals_api/spec/models/supplemental_claim_spec.rb

```diff
@@ -49,163 +50,248 @@
         }
       end
 
+      let(:sc_bad_issues_dates) do
+        described_class.new(form_data: form_data, auth_headers: default_auth_headers, api_version: 'V2')
+      end
+
       it 'creates an invalid record' do
-        expect(supplemental_claim.valid?).to be false
-        expect(supplemental_claim.errors.to_a.length).to eq 1
-        expect(supplemental_claim.errors.to_a.first.downcase).to include 'decisiondate'
-        expect(supplemental_claim.errors.to_a.first.downcase).to include 'past'
+        expect(sc_bad_issues_dates.valid?).to be false
+        expect(sc_bad_issues_dates.errors.to_a.length).to eq 1
+        expect(sc_bad_issues_dates.errors.to_a.first.downcase).to include 'decisiondate'
+        expect(sc_bad_issues_dates.errors.to_a.first.downcase).to include 'past'
       end
     end
   end
 
-  context 'headers' do
-    let(:auth_headers) { fixture_as_json 'valid_200995_headers.json', version: 'v2' }
-    let(:supplemental_claim) { create(:supplemental_claim) }
+  describe '#full_name' do
+    it { expect(sc_with_nvc.full_name).to eq 'joe b smart' }
```

Comment:
I think anywhere that the Veteran's info could be overridden by the NVC should have a test for both scenarios. We should keep around the veteran-only tests AND these NVC override tests.